stories/topics/proc-aap-aws-application-upgrade.adoc

```diff
@@ -0,0 +1,150 @@
+[id="proc-aap-aws-application-upgrade"]
+
+Tech Preview
+~~~~~~~~~~~~
+
+You can upgrade your existing {AAPonAWS} deployment to the newer version. The upgrade process covers upgrading of {HubName} and {ControllerName}. The upgrade process takes roughly the same amount of time as a {AAPonAWS} deployment install. Follow the steps below to ensure a smooth upgrade process.
+
+[NOTE]
+=====
+{AAPonAWS} supports sequential upgrades. All upgrades should be no more than one minor version behind what you are currently upgrading to. For example, in order to upgrade to {AAPonAWS} to 2.2.20230215, you must be on the first version of {AAPonAWS}.
+=====
+
+.Prerequisites
+* Linux or macOS system (where the `ansible-on-clouds-ops` container image will run)
+* Docker
+
+.Procedure
+. Create the prepare the environment and the configuration files.
+. Run the `ansible-on-clouds-ops` container to upgrade your {AAPonAWS} deployment.
+
+== Preparing the environment
+
+The upgrade process requires a few volumes to be mounted, so *prepare a fresh directory* to be used for this process.
```

Comment:
The upgrade process requires a few volumes to be mounted. Prepare a directory you can use for this purpose.